py3status/py3.py

```diff
@@ -40,6 +51,41 @@ def __repr__(self):
         return 'None'
 
 
+class HTTPResponse:
+    """
+    This class is like a simplified version of `urllib3.response.HTTPResponse`.
+
+    The HTTPResponse object provides status, data, and json attributes.
+
+    @TODO: Implement the header attribute
+    """
+
+    def __init__(self, resp_obj):
+        self._resp_obj = resp_obj
+        self._data = None
+        self._json = None
+
+    def __getattr__(self, name):
+        # If trying to access an attribute that doesn't exist in this class,
+        # try accessing that attribute from the response object.
+        return getattr(self._resp_obj, name)
```

Comment:
hmm,

I'd remove this.  the reason is that if you allow this you are effectively creating an api that you need to support forever.  By limiting the methods eg json() then it is easy to support them even it the internals change.  Also are responses from python2/3 100% identical?